2005-08-03  Dwayne Grant McConnell  <dgm69@us.ibm.com>

        * sysdeps/powerpc/powerpc32/Makefile (CFLAGS-initfini.s):
        Use +=, not =, to append.  Remove duplicate flags.
        * sysdeps/powerpc/powerpc64/Makefile (CFLAGS-initfini.s):
        Likewise.  Fix comments.

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/powerpc/powerpc32/Makefile,v
retrieving revision 1.5
retrieving revision 1.6
diff -u -r1.5 -r1.6
--- libc/sysdeps/powerpc/powerpc32/Makefile	2003/10/22 21:30:02	1.5
+++ libc/sysdeps/powerpc/powerpc32/Makefile	2005/08/04 21:35:43	1.6
@@ -21,7 +21,7 @@
 ifneq ($(elf),no)
 # The initfini generation code doesn't work in the presence of -fPIC, so
 # we use -fpic instead which is much better.
-CFLAGS-initfini.s = -g0 -fpic -O1
+CFLAGS-initfini.s += -fpic -O1
 
 # There is no benefit to using sdata for these objects, and the user
 # of the library should be able to control what goes into sdata.

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/powerpc/powerpc64/Makefile,v
retrieving revision 1.5
retrieving revision 1.6
diff -u -r1.5 -r1.6
--- libc/sysdeps/powerpc/powerpc64/Makefile	2004/10/06 22:05:14	1.5
+++ libc/sysdeps/powerpc/powerpc64/Makefile	2005/08/04 21:35:44	1.6
@@ -12,9 +12,9 @@
 
 ifeq ($(subdir),csu)
 ifneq ($(elf),no)
-# The initfini generation code doesn't work in the presence of -g1 or
-# higher, so we use -g0.
-CFLAGS-initfini.s = -g0 -fpic -O1
+# The initfini generation code doesn't work in the presence of -fPIC, so
+# we use -fpic instead which is much better.
+CFLAGS-initfini.s += -fpic -O1
 endif
 endif
 
2006-04-26  Mike Frysinger  <vapier@gentoo.org>

        * sysdeps/powerpc/Makefile (CFLAGS-pt-initfini.s): Use +=, not =, to
        append.  Remove duplicate flags.

===================================================================
RCS file: /cvs/glibc/linuxthreads/linuxthreads/sysdeps/powerpc/Makefile,v
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- linuxthreads/linuxthreads/sysdeps/powerpc/Makefile	2003/02/17 20:33:17	1.2
+++ linuxthreads/linuxthreads/sysdeps/powerpc/Makefile	2006/04/26 15:46:42	1.3
@@ -1,6 +1,6 @@
 ifeq ($(subdir):$(elf),linuxthreads:yes)
 # See CFLAGS-initfini.s above; this is the same code.
-CFLAGS-pt-initfini.s = -g0 -fpic -O1
+CFLAGS-pt-initfini.s += -fpic -O1
 endif
 ifeq ($(subdir),csu)
 gen-as-const-headers += tcb-offsets.sym
